### DISCUSSION TITLE
Save Copy As...

### DISCUSSION THREAD

**Discussion by booraroom** [atom/atom#22296] [Q&A]
Hi all, my first post here, hope you are all well. I wasn't sure in which category to put this question, so I started with Q&A. I did some preliminary searching but didn't find anything related to this (or my search strings weren't up to the task).

Is there an extension or enhancement to support the `Save Copy As...` concept in Atom? It's similar to `Save As...`, except after saving the file it leaves the editor window pointing to the original file (i.e. subsequent `Save`s would overwrite the original, leaving the new copy intact).

For example: say I'm editing `index.html`; I choose `Save Copy As...` and save it as `index_orig.html`; I continue editing then choose `Save`, which overwrites `index.html` and leaves `index_orig.html` intact.

If this doesn't currently exist, then my next question (before I invest the time) is:  would this be a useful addition to the Atom core, or do you feel this would be better suited for an extension?

Thanks for your time and attention.

**Comment by UziTech** [comment]
> would this be a useful addition to the Atom core, or do you feel this would be better suited for an extension?

If you want it in Atom it would most likely be faster to create an extension. Getting new non-essential features into Atom core takes a few years on average.
